stable/wordpress/README.md

```diff
@@ -138,7 +138,7 @@ Note also if you disable MariaDB per above you MUST supply values for the `exter
 This chart provides support for ingress resources. If you have an
 ingress controller installed on your cluster, such as [nginx-ingress](https://kubeapps.com/charts/stable/nginx-ingress)
 or [traefik](https://kubeapps.com/charts/stable/traefik) you can utilize
-the ingress controller to service your WordPress application.
+the ingress controller to the service of your WordPress application.
```

Comment:
I would use " to serve your WordPress application"

Reply:
Great idea! @javsalgar 
It's done.